src/plugins/Toast/Toast.ts

```diff
@@ -1,74 +1,107 @@
-import { App, createApp, InjectionKey, Component } from 'vue'
+import { App, createApp, reactive, Plugin } from 'vue'
 import ToastContainer from '@/components/Toast/ToastContainer.vue'
 
-declare module '@vue/runtime-core' {
-  export interface ComponentCustomProperties {
-    $toast: {
-      add: (options: ToastOptions) => MountedElement
-    }
-  }
-}
+type ToastType = 'default' | 'success' | 'error'
 
-export type MountedElement = {
-  remove(): any
+type Toast = ToastOptions & {
+  id: number,
+  dismiss: () => void
 }
 
 type ToastOptions = {
-  component?: Component
-  content?: string
+  message: string,
+  type?: ToastType,
   timeout?: number
   dismissable?: boolean
-  type?: string
-  classList?: string[]
```

Comment:
Were `component` and `classList` unused and that's why they're being removed?

Reply:
correct. Nowhere in orion do we actually use those, so I'm removing them in favor of not engineering solutions to problems we don't have.